netmap/netmap_contract.go

```diff
@@ -95,8 +99,8 @@ func _deploy(data any, isUpdate bool) {
 
 	var args = data.(struct {
 		notaryDisabled bool
-		addrBalance    interop.Hash160
-		addrContainer  interop.Hash160
+		addrBalance    interop.Hash160 // Balance contract not used legacy
+		addrContainer  interop.Hash160 // Container contract not used legacy
```

Comment:
BTW, if these two fields are unused, then you can explicitly mark them as `_`.

Reply:
fixed